src/bin/21_descriptor_layout_and_buffer.rs

```diff
@@ -0,0 +1,727 @@
+#![feature(duration_as_u128)]
```

Comment:
```#![feature] may not be used on the stable release channel```
-> no unstable features please unless there's a very good reason.

Reply:
Just seeing these comments now. I'll update this to not use features tomorrow.